ui_tests/pages/inventory_page.py

```diff
@@ -0,0 +1,336 @@
+import time
+
+import pages.helper as helper
+import selenium.common.exceptions
+from logger.logger import Logger
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+from webdriver.webriver_factory import WebDriverFactory
+
+logger = Logger().get_logger()
+driver = WebDriverFactory.get_driver()
+
+
+class InventoryPage:
+    def checkIfInventoryTableIsDisplayed(self):
+        logger.info("Check if groups page is displayed")
```

Comment:
change log message to "Check if inventory page is displayed"

Reply:
done